rssfeed/models.py

```diff
@@ -1,53 +1,50 @@
 from django.db import models
+from django.utils.translation import ugettext_lazy as _
 
 
 class Feed(models.Model):
     title = models.CharField(max_length=2000, blank=True, null=True)
-    xml_url = models.CharField(max_length=255, unique=True)
-    link = models.CharField(max_length=2000, blank=True, null=True)
+    url = models.CharField(max_length=2000, unique=True)
     description = models.TextField(blank=True, null=True)
-    published_time = models.DateTimeField(blank=True, null=True)
-    last_polled_time = models.DateTimeField(blank=True, null=True)
+    link = models.CharField(
+        max_length=255,
+        blank=True,
+        null=True,
+        db_index=True
+    )
+    published = models.DateTimeField(blank=True, null=True)
+    last_polled = models.DateTimeField(blank=True, null=True)
+    image = models.ImageField(max_length=2000, null=True)
 
     class Meta:
-        verbose_name = "Feed"
-        verbose_name_plural = "Feeds"
+        verbose_name = _("Feed")
```

Comment:
I'm not exactly sure how they are translated, but ugettext_lazy translates the strings in a lazy fashion, which means, every time you access the name of an attribute on a model the string will be newly translated. 

Am I missing something @rijkstofberg ?